scripts/split-paired-reads.py

```diff
@@ -72,11 +73,13 @@ def get_parser():
                         'directory if necessary')
 
     parser.add_argument('-1', '--output-first', metavar='output_first',
+                        type=argparse.FileType('w'),
                         default=None, help='Output "left" reads to this '
-                        'file', type=argparse.FileType('w'))
```

Comment:
Oh, I know what happened. I took it out and added it back it, but just flipped where it was.

Whoops.